ledger/internal/cow.go

```diff
@@ -128,6 +128,21 @@ func (cb *roundCowState) deltas() ledgercore.StateDelta {
 			}
 		}
 	}
+
+	// Populate old values by looking through parent
+	for key, value := range cb.mods.KvMods {
+		old, ok, err := cb.lookupParent.kvGet(key) // Because of how boxes are prefetched, value will be cached
+		if err != nil {
+			panic(key)
```

Comment:
let's provide a better message

Reply:
Do people like this approach?
```
panic(fmt.Errorf("Error looking up %v : %w", key, err))
```
I guess it doesn't really accomplish anything that
```
panic(fmt.Sprintf("Error looking up %v : %s", key, err.String()))
```
does not. But I suppose it's more concise. Does it seem strange to hand a error value to `panic()`?